cmd/adhoc/cmd/root.go

```diff
@@ -24,6 +28,30 @@ func init() {
 	)
 }
 
+func readFile() {
+	// Setting up file formatting
+	viper.SetConfigFile("syncron.yaml")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".")
+	// Reading from file
+	err := viper.ReadInConfig()
+	if err != nil {
+		logrus.Error("Config file not found.\n")
+		logrus.Info("Make sure syncron.yaml is in the correct folder. \n")
+		logrus.Info("Syncron looks for the config file on: \n")
+		logrus.Info("	- Root of the repository \n")
+		os.Exit(1)
```

Comment:
I would return here the error you got, or a rewrite of it, and let the app close on its own. If down the lane you have some cleanup to do before closing syncron, it will free you from a headache or two.